actioncable/lib/action_cable/connection/base.rb

```diff
@@ -126,7 +126,8 @@ def on_message(message) # :nodoc:
       end
 
       def on_error(message) # :nodoc:
-        # ignore
+        # log errors to make diagnosing socket errors easier
+        logger.error "WebSocket error occurred: #{message}"
```

Comment:
But when were you looking for it? In development time or production time? I feel that this information is not important enough to be reported in the error log level since this seems more like an information to help to debug a problem, not an actual error.